src/hotspot/cpu/ppc/templateInterpreterGenerator_ppc.cpp

```diff
@@ -1030,7 +1030,10 @@ void TemplateInterpreterGenerator::generate_fixed_frame(bool native_call, Regist
   // Store values.
   __ std(R19_method, _ijava_state_neg(method), R1_SP);
   __ std(Rmirror, _ijava_state_neg(mirror), R1_SP);
-  __ std(R18_locals, _ijava_state_neg(locals), R1_SP);
+  __ sub(R12_scratch2, R18_locals, R1_SP);
+  __ srdi(R12_scratch2, R12_scratch2, Interpreter::logStackElementSize);
+  // Now &fp()[R12_scratch2] == R18_locals
```

Comment:
I think this comment makes no sense. Please remove it. Otherwise, PPC64 code LGTM.

Reply:
My intention was to show how the contents of R18_locals (the absolute address to locals) can be recreated by using R12_scratch2 as an index from the frame pointer. Before we stored R18_locals in the stackframe, now we store R12_scratch2. Does that make more sense?

Reply:
The code uses &sp()[R12_scratch2]. I think your comment is confusing. Please remove it or write something comprehensive. E.g. save R18_locals in compressed format relative to the frame's SP.

Reply:
See what you mean, the R1_SP being the frame pointer is confusing. So how about changing the comment into:
"// Store relativized R18_locals, see frame::interpreter_frame_locals()."
Would that work for you? If not I'll just remove it.

Reply:
That's fine. Thanks!